src/App.js

```diff
@@ -10,16 +10,27 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies
+      movies: movieData.movies,
+      movieShow: []
     }
   }
 
+  goToIndex = () => {
```

Comment:
Same with the naming here - goToIndex doesn't really describe what this is going. Could clearMovies() work instead?